flake.nix

```diff
@@ -28,10 +41,12 @@
     , fenix
     , naersk
     , nix
+    , determinate
     , ...
     } @ inputs:
     let
-      supportedSystems = [ "i686-linux" "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
+      supportedSystems = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
+      systemsSupportedByDeterminateNixd = [  ]; # avoid refs to detsys nixd for now
```

Comment:
@mkenigs This is a new change in the most recent force-push. 

While working on flaky coffee-shop WiFi I noticed a build time out while trying to go out to download a daemon binary. I thought about also just excluding the flake input, but I don't want to fiddle with something that probably implies a big change to the flake lock if we can just not actually pull it in?

Reply:
Hehe.

I wanted to leave this comment on the diff of just my force-push, but GH doesn't give us inline comments there and I didn't pay enough attention to the diff when I came to this copy to comment :)

You can't tell _here_, but my change was just to empty the `systemsSupportedByDeterminateNixd` list (which the upstream side of the merge added here as `systemsSupportedByDeterminateNixd = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];`. 

A better ref: https://github.com/NixOS/experimental-nix-installer/compare/43416a75ca963d93ef6fed1c8539a09c65d41f07..b8b7ace74072d0a32b5d12a3c83b8fa7968de8cd#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R49

Reply:
Ah okay. Looks good 👍